quarkus-workshop-super-heroes/docs/src/docs/asciidoc/core-appendix/appendix-installing-curl.adoc

```diff
@@ -1,67 +1,6 @@
-[[appendix-installing-curl]]
+[[appendix-installing-jq]]
```

Comment:
I don't think installing jq should be in the appendix, because it's required. (Unlike curl, which is always-there and not required except on a few linux distros.)